cypress/e2e/po/pages/users-and-auth/users-and-auth.po.ts

```diff
@@ -0,0 +1,78 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import CheckboxInputPo from '~/cypress/e2e/po/components/checkbox-input.po';
+
+export default class UsersAndAuthPo extends PagePo {
+  static url: string = '/c/_/auth/management.cattle.io.user'
```

Comment:
This needs to match the pattern below. It's not an issue here, but will save someone getting confused if they write a test that goes via another cluster (the blank cluster will change to the last one visited)

```
  private static createPath(clusterId: string) {
    return `/c/${ clusterId }/manager/provisioning.cattle.io.cluster`;
  }

  static goTo(clusterId: string): Cypress.Chainable<Cypress.AUTWindow> {
    return super.goTo(ClusterManagerListPagePo.createPath(clusterId));
  }

  constructor(clusterId: string = '_') {
    super(ClusterManagerListPagePo.createPath(clusterId));
  }
```

Reply:
pattern matched